pkg/util/spanlogger/spanlogger.go

```diff
@@ -23,15 +22,11 @@ var defaultTenantResolver = tenant.NewMultiResolver()
 // SpanLogger unifies tracing and logging, to reduce repetition.
 type SpanLogger = spanlogger.SpanLogger
 
-// New makes a new SpanLogger with a log.Logger to send logs to. The provided context will have the logger attached
-// to it and can be retrieved with FromContext.
-func New(ctx context.Context, method string, kvps ...interface{}) (*SpanLogger, context.Context) {
-	return spanlogger.New(ctx, util_log.Logger, method, defaultTenantResolver, kvps...)
-}
-
-// NewWithLogger is like New but allows to pass a logger.
-func NewWithLogger(ctx context.Context, logger log.Logger, method string, kvps ...interface{}) (*SpanLogger, context.Context) {
-	return spanlogger.New(ctx, logger, method, defaultTenantResolver, kvps...)
+// New makes a new SpanLogger with a log.Logger to send logs to.
+// It starts a new OTEL span with the method name provided using the tracer provided.
```

Comment:
[Nit] OTel isn't an acronym:
```suggestion
// It starts a new OTel span with the method name provided using the tracer provided.
```